src/frame/endpoints/UserApiEndpoints.ts

```diff
@@ -179,6 +179,21 @@ const userUpdate = (userID: string, userPrivateKey?: PrivateKey<Uint8Array>, sta
     errorCode: ErrorCodes.USER_UPDATE_REQUEST_FAILURE,
 });
 
+const userKeyUpdateApi = (userID: string, userPrivateKey: PrivateKey<Uint8Array>, augmentationFactor: AugmentationFactor, userKeyId: number): RequestMeta => ({
+    url: `users/${encodeURIComponent(userID)}/keys/${userKeyId}`,
+    options: {
+        method: "PUT",
+        headers: {
+            "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+            userPrivateKey,
+            augmentationFactor,
+        }),
+    },
+    errorCode: ErrorCodes.USER_UPDATE_REQUEST_FAILURE,
```

Comment:
I think it would be a good idea to add a unique error code for this since it's a different URL we're hitting than normal use update. So add something like `USER_UPDATE_KEY_REQUEST_FAILURE`.